tests/foreman/longrun/test_inc_updates.py

```diff
@@ -0,0 +1,301 @@
+"""Tests for the Incremental Update feature"""
+
+from datetime import date, timedelta
+from fauxfactory import gen_alpha
+from nailgun import entity_mixins
+from nailgun.entities import (
+    ActivationKey, ContentView, ContentViewFilterRule, ContentViewVersion,
+    Errata, ErratumContentViewFilter, HostCollection, LifecycleEnvironment,
+    Organization, Repository, Subscription, System
+)
+from robottelo.api.utils import (
+    enable_rhrepo_and_fetchid, promote, upload_manifest
+)
+from robottelo.cli.contentview import ContentView as ContentViewCLI
+from robottelo.common import manifests
+from robottelo.common.constants import PRD_SETS
+from robottelo.common.decorators import run_only_on, skip_if_bug_open
+from robottelo.vm import VirtualMachine
+from unittest import TestCase
+
+
+@run_only_on('sat')
+class TestIncrementalUpdate(TestCase):
+    """Tests for the Incremental Update feature"""
+
+    @classmethod
+    def setUpClass(cls):
+        """Creates all the pre-requisites for the Incremental updates test"""
+        # Step 1 - Create a new Organization
+        cls.org = Organization(name=gen_alpha()).create()
+
+        # Step 2 - Create two life cycle environments - DEV, QE
+        cls.dev_lce = LifecycleEnvironment(
+            name='DEV',
+            organization=cls.org
+        ).create()
+        cls.qe_lce = LifecycleEnvironment(
+            name='QE',
+            prior=cls.dev_lce,
+            organization=cls.org
+        ).create()
+
+        # Step 3: Upload manifest
+        with open(manifests.clone(), 'rb') as manifest:
+            upload_manifest(cls.org.id, manifest)
+
+        # Step 4: Enable repositories - 6Server and rhel6 sat6tools
+        rhel_66_repo_id = enable_rhrepo_and_fetchid(
+            basearch=PRD_SETS['rhel_66']['arch'],
+            org_id=cls.org.id,
+            product=PRD_SETS['rhel_66']['product'],
+            repo=PRD_SETS['rhel_66']['reponame'],
+            reposet=PRD_SETS['rhel_66']['reposet'],
+            releasever=PRD_SETS['rhel_66']['releasever']
```

Comment:
Manual or automation-tools that's just a suggestion to make things more reliable, whatever suits the best, here it is automation-tools of-course.

a) sync rhel6 content takes approx 2 hours.
b) What I am suggesting is sync this huge content in Org "a"  before the automation is started, may be via automation-tools.
c) No changes are required any longer in this PR.
d) Then when this test is run, a new org gets created Org "b", this time while running the test the test won't be running for 2+ hours but may be finish in minutes and thus be very reliable
Why? Because, this time pulp would not re-download all of the 26GB content, but just check for the "rhel6 metadata" and it would finish saying "no packages" to download.

Benefit:
a) This would help collect more reliable data on jenkins, that's the major benefit here. Test Pass probability would be high, as the focus is not on syncing content 25+GB but rather on incremental updates.
b) If we don't separate it out, I think it may cause severe reliability issues in syncing 25GB+ content.
c) It would immensely benefit if we trigger it using "hammer cli"  commands, which could be added to automation-tools as a function and could be run only whenever it's needed.